opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -356,6 +436,8 @@ class SimulatorFullyImplicitBlackoilEbos
             OpmLog::debug(msg);
         }
 
+        handleSave(timer);
```

Comment:
This is okay as a start, but ideally this would be triggered at a higher level to enable integrating with the `SAVE` keyword.

Reply:
i'm not sure if SAVE support for require reorganization (we have schedule and whatnot here), however i suspect the member variables might go into restartconfig. but sure, i'm not married to where this is called in any way.